### DISCUSSION TITLE
Minimize Fonts ttfonts file size

### DISCUSSION THREAD

**Discussion by kevinjavitz** [mpdf/mpdf#1922] [Q&A]
Hi,

Since the vendor folder has to go in the WordPress plugin folder in order to load PDF for my use case, is there a safe way to minimize the fonts in the ttfonts folder? Are there some fonts that have to stay there, and some that are extra? It is about 86mb of fonts would be nice to reduce it. 

In a little test I did it seems only the DejaVu* series of fonts is really needed. I'm only generating roman family fonts currently in the pdfs.

**Comment by finwe** [comment]
You don't need any of the fonts in the folder unless you really use them.

Modifying contents of the vendor folder is another thing.

See also: https://github.com/mpdf/mpdf/issues/552 https://github.com/mpdf/mpdf/pull/841
